src/OpenLoco/Widget.cpp

```diff
@@ -781,6 +781,52 @@ namespace OpenLoco::Ui::Widget
         Gfx::fillRect(dpi, left, top, right, bottom, Colour::getShade(Colour::black, 5));
     }
 
+    void drawGroupbox(Gfx::drawpixelinfo_t* const context, window* const window, widget_t* widget)
+    {
+        const uint8_t colour = window->colours[widget->colour] & 0x7F;
+        int32_t l = window->x + widget->left + 5;
+        int32_t t = window->y + widget->top;
+        int32_t r = window->x + widget->right;
+        int32_t b = window->y + widget->bottom;
+        int32_t textEndPos = l;
```

Comment:
This doesn't look to follow loco at all have you just repurposed the wt_16?

Reply:
Yes. As above, I couldn't find groupboxes in the Locomotion widget code.